src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -1609,10 +1620,19 @@ private void LoadTimeSeriesPolicyRunnerConfigurations()
                         }
                     }
 
-                    if (ServerStore.LicenseManager.HasHighlyAvailableTasks() == false)
+                    if (serverStore.LicenseManager.HasHighlyAvailableTasks() == false)
                     {
                         // can't redistribute, keep it on the original node
-                        RaiseAlertIfNecessary(databaseTopology, configuration, lastResponsibleNode);
+                        if (databaseTopology.Count > 1 &&
```

Comment:
Why was it changed?

Reply:
Returned to the previous implementation, adding a check for `notificationCenter` not being null.